build/repo.targets

```diff
@@ -1,9 +1,13 @@
 <Project>
   <ItemGroup>
     <Scenarios Include="-n Plaintext -o KestrelHttpServer@dev --kestrelThreadCount 1" />
+    <Scenarios Include="-n Plaintext -o KestrelHttpServer@dev --kestrelThreadCount 1 --kestrelThreadPoolDispatch false" />
```

Comment:
Curious: does the thread dispatching choice matter when the thread count is `1`?